assets/wizards/performance/views/intro/index.js

```diff
@@ -27,11 +27,12 @@ class Intro extends Component {
 			<div className='muriel-wizardScreen muriel-wizardScreen__no-background'>
```

Comment:
Performance Wizard is the only one I could find where "Back To Dashboard" button is centered below the Card. Is this intentional?

<img width="1552" alt="Screen Shot 2019-12-12 at 8 56 05 AM" src="https://user-images.githubusercontent.com/1477002/70718016-7412b080-1cbd-11ea-839b-91496fddfd12.png">

Reply:
It wasn't. Should be fixed with 67bc7be